samples/compose-samples/src/main/AndroidManifest.xml

```diff
@@ -2,14 +2,16 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+  <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
How come?

Reply:
New requirement to request this permission if your app posts any notifications. Good point that I didn't really look into why this was being linted and if there was a way of specifying that we _don't_ post notifications. I kind of thought that, hey, might as well just include it. Too messy?

Reply:
So this is a branch new permission that only applies to API 33+ installs. On those you need to request the permission at runtime. There will be follow-on work for this. Alternative would be to leave it out and disable the Android Lint setting for this? I think this is likely the least intrusive way forward?

What will catch us is when we start running on API 33 devices. I will create an issue to track that.

Reply:
It looks for code that uses Notifications, but this can be in classes from dependent libraries - classes which aren't necessarily used. This is only a warning, but since we have warning=error it fails the build.

I'm going to take these out and take a lint warnings baseline, then use that going forward.

Reply:
Turns out I don't think we need `POST_NOTIFICATIONS` once i finished updating some transitive dependency libraries (this isn't in the lint baseline). Obviously one of the libraries had to update for Android 13 as well.